dns/py/runner.py

```diff
@@ -459,6 +463,16 @@ def _wait_for_status(self, active):
 
       _log.info('DNS is up')
 
+  def _get_pod(self, label, output='custom-columns=:metadata.name'):
+    code, client_pods, err = self._kubectl(None, 'get', 'pods', '-l', label, '--no-headers', '-o',
```

Comment:
The label selector looks identical to what exists in the code already. "app=dns-perf-client" What is this fixing?

Reply:
ignore this.. I found the issue in one exec call which was hardcoding client name.